core/src/main/java/io/questdb/cutlass/http/processors/JsonQueryProcessorState.java

```diff
@@ -230,6 +230,14 @@ public void logExecuteNew() {
                 $(']').$();
     }
 
+    public void logTimings() {
+        info().$("timings ").
+                $("[compiler: ").$(compilerNanos).
+                $(", count: ").$(recordCountNanos).
+                $(", execute: ").$(nanosecondClock.getTicks() - executeStartNanos).
+                $(']').$();
```

Comment:
we need more here, at least fd. It would be impossible to match timings to a query without these that. Perhaps logging sql text again will help too

Reply:
sure, checking now

Reply:
fd was already there, info() has the fd

Reply:
will look like this now:
`2020-06-24T15:19:38.187565Z I i.q.c.h.p.JsonQueryProcessorState [17] timings [compiler: 7825269, count: 0, execute: 8165024, q=select distinct pair from x ]`